docker-compose.yml

```diff
@@ -148,16 +171,32 @@ services:
 
   kong-migrations:
     image: ${kong}
-    container_name: kong-migration
-    hostname: kong-migration
+    container_name: kong-migrations
     networks:
       edgex-network:
         aliases:
-          - kong-migration
+          - kong-migrations
     environment:
       - 'KONG_DATABASE=postgres'
       - 'KONG_PG_HOST=kong-db'
-    command: "kong migrations bootstrap"
+    # JQ broken in ubuntu image due to libc dependency
+    # https://github.com/edgexfoundry/docker-edgex-consul/issues/3
+    # sleep 30s instead of checking consul for DB healthy
```

Comment:
@cloudxxx8, @michaelestrin 

Please note  for https://github.com/edgexfoundry/blackbox-testing/issues/295 that our race condition checking for blackbox-testing is currently broken due to using jq from consul which is libc-based, and we are needed to use the ubuntu kong for blackbox testing.  This can be fixed with https://github.com/edgexfoundry/docker-edgex-consul/issues/3, but the implications are that we probably need to use the apline-based kong images for x86 and the ubuntu-based kong images for aarch64.  The aarch64 will need its dockerfile customized to include this workaround and own't be able to use the developer-scripts version directly.

Reply:
We have [an issue](https://github.com/edgexfoundry/developer-scripts/issues/154) to create arm64 docker-compose files in developer-scripts.  Am I receiving correctly that we'll need to have a customized version of those arm64 files for blackbox testing?  @bnevis-i

Reply:
@michaelestrin I have fresh eyes today and will give this another look today and try to determine if the above statement is true or not under what conditions.

Reply:
@michaelestrin 

> We have an issue to create arm64 docker-compose files in developer-scripts. Am I receiving correctly that we'll need to have a customized version of those arm64 files for blackbox testing?

Short answer: no (to the best of my knowledge)

The delta are:
  - Most containers will need to switch to ubuntu-based images for aarch64 compatibility.
  - Besides that, aarch64 will need to rely on hard-coded delays INSTEAD OF dependency checker for kong-migrations and kong UNTIL we have a golang-version of the dependency checker or a statically-linked jq.  This delta is obviously more complicated than simply switching container version.

More detail:

In Fuji:
 - The docker-compose scripts between developer-scripts and blackbox-testing do not have feature parity.
 - Docker developer-scripts run because Ian implemented depenency checking hooks in all of the critical security-related containers.
 - The blackbox-testing scripts run because of the hard-coded delays between starting the services.
 - The developer-scripts docker-compose scripts use kong(-alpine) images (MUSL-based)
 - The blackbox-testing docker-compose scripts use kong-centos (glibc-based) for x64 because that is all that works on the VM.
 - The blackbox-testing docker-compose scripts use kong-ubuntu (glibc-based) for aarch64 because that is all the works on the emulator.

In Geneva:
 - The VM/emulator issues continue to exist (only images that work for Kong are glibc-based)
 - The kong images don't have a glibc version of jq installed by default
 - Until we have a golang-version of the dependency checker or a statically-linked jq, the dependency-checking scripts on the Kong containers will choke on the non-alpine Kong.  It will choke on the non-alpine Kong regardless of whether it is developer-scripts or blackbox-testing.